@xen-orchestra/lite/src/components/ui/UiStatusPill.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <UiIcon :icon class="icon" :class="props.state" />
+</template>
+
+<script lang="ts" setup>
+import UiIcon from '@/components/ui/icon/UiIcon.vue'
+import { computed } from 'vue'
+import { faCheckCircle, faCircleMinus, faCircleXmark } from '@fortawesome/free-solid-svg-icons'
+import type { Color } from '@/types'
+
+const props = defineProps<{
+  state: Color
```

Comment:
You can add the typing with values used in the design system.
```suggestion
  state: 'success' | 'partial' | 'failure'
```